src/components/modify-todo-box.tsx

```diff
@@ -0,0 +1,97 @@
+import { SyntheticEvent, useEffect, useState } from "react";
+import { useParams, useNavigate } from "react-router-dom";
+import useTodos from "@/hooks/useTodos";
+import { Button } from "@/components/ui/button";
+import { Card } from "@/components/ui/card";
+import { Label } from "@/components/ui/label";
+import { Textarea } from "@/components/ui/textarea";
+import { Input } from "@/components/ui/input";
+import { Todo } from "@/types/todo-type";
+
+const ModifyTodoBox = () => {
+  const params = useParams();
+  const navigate = useNavigate();
+  const [todo, setTodo] = useState<Todo | null>(null);
+  const { isLoading, getTodoById, modifyTodo, deleteTodo } = useTodos();
+
+  const handleUpdate = (e: SyntheticEvent) => {
+    e.preventDefault();
+
+    const form = e.target as HTMLFormElement;
+    const titleInput = form.elements.namedItem("title") as HTMLInputElement;
+    const descInput = form.elements.namedItem(
+      "description"
+    ) as HTMLInputElement;
+
+    if (e.target) {
+      const newTodo: Omit<Todo, "id" | "createdAt" | "completed"> = {
+        title: titleInput.value ?? "",
+        description: descInput.value ?? "",
+      };
+
+      modifyTodo(params.id!, newTodo);
+      alert("성공적으로 수정되었습니다.");
+      navigate("/");
+    } else {
+      alert("잘못된 요청입니다.");
+    }
+  };
+
+  const handleDelete = (e: SyntheticEvent) => {
+    e.preventDefault();
+
+    deleteTodo(params.id!);
+    alert("성공적으로 삭제되었습니다.");
+    navigate("/");
+  };
+
+  useEffect(() => {
+    if (isLoading) {
+      return;
+    }
+    const initialTodo = getTodoById(params.id ?? "");
+    if (!initialTodo) {
+      alert("잘못된 요청입니다.");
+      navigate("/");
+    } else {
+      setTodo(initialTodo);
+    }
+  }, [isLoading]);
+
+  if (isLoading) return <div>Loading...</div>;
```

Comment:
L49-L51의 early return문과 동일하게 아래처럼 컨벤션을 맞추면 어떨까요?
```suggestion
  if (isLoading) {
     return <div>Loading...</div>;
  }
```

Reply:
[f087e72](https://github.com/f-lab-edu/benee-todo/pull/2/commits/f087e72eca70cea37c605793acbdc8b2bce2c330) 에서 수정하였습니다.